### PR TITLE
updated readme

### LINKED ISSUE #198: Update main README
## Description
Add more description and instructions to run

## Required Files
- README.md

## User Story
As a technical person outside of the project I need to be able to get the product up and running from the instructions provided

### PR DESCRIPTION
**Pull request by shindym**
- Updated main readme to include more details about the project.

Closes #198

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,77 @@
+# Games Tracker Project
 ## Description
+> This Project is responsible for helping users to keep up with the latest game releases as well as understand the latest gaming trends.
 
-> This Project is responsible for helping users to keep up with the latest releases as well as understand latest gaming trends.
-## Stakeholder Requirements
+### Stakeholder Requirements
 -  `Gamers` - know what the latests game releases are.
 - `Developers`- understand the market more effectively.
```
Comment (hhm970) [inline README.md:7]: I think we may have to include a 3rd stakeholder to reflect our presentation, though this is something we will discuss as a group separately

File: README.md [hunk 2]
```diff
@@ -1,59 +1,77 @@
...
   cd terraform
   terraform init
   terraform apply
   ```
 
-## Files and Folder Structure Explained
+> [!NOTE]  
+> The ECR repositories were made via the AWS console. In order to be able to run the terraform script you must create the following ECR repositories listed below before.
+
+#### ECR Repositories
```
Comment (hhm970) [inline README.md:50]: Good detail, I didn't even think of that

File: README.md [hunk 3]
```diff
@@ -1,59 +1,77 @@
...
+- c10-games-dashboard
+- c10-games-db-load
+- c10-games-epic-extract
+- c10-games-gog-scrape
+- c10-games-steam-scrape
+- c10-games-weekly-report
+
+## Folder Structure Explained
```
Comment (hhm970) [inline README.md:58]: Good holistic explanation of the directories

**Review by hhm970 — APPROVED** [review]
I am content with this README.md, as it summarises all of the individual folders' purposes, and has all necessary prerequisites

**Review by annalisev — APPROVED** [review]
Nice changes, good detailed instructions for setup!

**Pull request merged**